.github/scripts/deploy_dbt_model_dependencies.sh

```diff
@@ -147,12 +147,12 @@ while read -r item; do
     subdirectory_name="${model_identifier}/"
     mkdir -p "$subdirectory_name"
     echo "Installing dependencies from $requirements_filename into $subdirectory_name"
-    pip install -t "$subdirectory_name" -r "$requirements_filename"
+    pip install -t "$subdirectory_name" -r "$requirements_filename" --no-deps
```

Comment:
The `--no-deps` flag gives us more fine-grained control over the dependencies that get installed, which is important because some more complex dependencies like `numpy` and `pandas` do not seem to install properly when installed into a target directory and then bundled into a zipfile. (`assesspy` depends on these two packages, hence the change as part of this PR.) It may be worth future investigation to see if there's just a trick that I'm missing for installing these kinds of packages, but in the meantime we can fall back to the prebuilt versions of these packages where necessary by avoiding installation of our dependencies' dependencies using the `--no-deps` flag.